assignment-client/src/octree/OctreeServer.cpp

```diff
@@ -1344,6 +1344,11 @@ QString OctreeServer::getUptime() {
     return formattedUptime;
 }
 
+double OctreeServer::getUptimeSeconds()
+{
+    return (usecTimestampNow() - _startedUSecs) / 1000000.0;
```

Comment:
what is the 1000000.0? seems like a magic number. If its `MICRO_SECONDS_IN_SECOND` perhaps define a constant with that name